vue/src/TagmanagerTrackingCode/TagmanagerTrackingCode.vue

```diff
@@ -36,7 +36,7 @@ export default defineComponent({
     TrackingCodeCommon,
   },
   directives: {
-    SelectOnFocus,
+    CopyToClipboard,
```

Comment:
Seems unused and can be removed, as there is no other change in that file.

Reply:
I replaced any existing use of `SelectOnFocus`, but didn't check if it was actually in use. Removed.